doc/changes/latest.rst

```diff
@@ -32,5 +32,5 @@ Changes
 -------
 
 - The documentation for :func:`~image.threshold_img` has been improved, with more information about which voxels are set to 0 and which ones keep their original values (:gh:`3485` by `Rémi Gau`_).
-
 - Modules :mod:`~nilearn.decomposition` and :mod:`~nilearn.decoding` code and docstrings have been reformatted using black. Changes resulted in improved readability overall and increased consistency (:gh:`3491` and :gh:`3484` by `Rémi Gau`_).
+- The behavior of :func:`fetch_atlas_craddock_2012`, :func:`fetch_atlas_smith_2009` and :func:`fetch_atlas_basc_multiscale_2015` is updated with their new parameters to return one map along with a deprecation cycle (:gh:`3353` by `Ahmad Chamma`_)
```

Comment:
Hello @ymzayek, thank you for your suggestions! I updated all the required code/tests. 
As for the ```latest.rst```, you mean adding a new latest file instead of this one?

Reply:
@achamma723 no it will be the same file but the prerelease changes (everything under the header `0.10.1rc1` will go to the bottom and new sections will be made under the header `0.10.1.dev`. So you would need to rebase on main and put this line under the new "Changes" section. If it's confusing, I could also do it myself so no worries!

Reply:
@achamma723 can you now pull from upstream main and rebase this pr on main?